extensions/sticky/extend.php

```diff
@@ -31,6 +31,9 @@
 
     new Extend\Locales(__DIR__.'/locale'),
 
+    (new Extend\Settings())
+        ->serializeToForum('excerptDisplayEnabled', 'flarum-sticky.config.enable_display_excerpt', 'boolval'),
```

Comment:
let's make it true by default so that the behavior is not changed from 1.x

let's also change the key from `flarum-sticky.config.enable_display_excerpt` to the conventional `flarum-sticky.enable_display_excerpt`

```suggestion
        ->serializeToForum('excerptDisplayEnabled', 'flarum-sticky.enable_display_excerpt', 'boolval')
        ->default('flarum-sticky.enable_display_excerpt', true),
```